code/api/src/modules/user/mutations.js

```diff
@@ -3,7 +3,7 @@ import { GraphQLString, GraphQLInt } from 'graphql'
 
 // App Imports
 import { UserType } from './types'
-import { create, remove } from './resolvers'
+import { create, remove, update } from './resolvers'
```

Comment:
User story actually asks for POST image. But I think we GraphQL, we can use update for post or patch?

cc: @priyapower